src/routines/get_results.jl

```diff
@@ -60,19 +64,7 @@ function get_model_duals(op_m::OperationModel, cons::Vector{Symbol})
 
 end
 
-function write_model_result(op_m::OperationModel, path::String)
```

Comment:
Is this the new `_write_variable`  code?

Reply:
this code was already in write model, it *I believe* is what is being used to right directly from the multi-day simulations to feather files, so I haven't touched it.